[flatbuffer] Use 7b params in Llama example

lucylq opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #1407
* #1406

Differential Revision: [D52137894](https://our.internmc.facebook.com/intern/diff/D52137894)

lucylq committed:
use 7b params

[ghstack-poisoned]

lucylq committed:
Update on "use 7b params"




[ghstack-poisoned]

iseeyuan commented on examples/models/llama2/model.py:
```diff
@@ -211,9 +211,9 @@ def __init__(self):
-        self.model_.load_state_dict(
-            checkpoint, strict=False
-        )  # self.model_ = Transformer(gptconf)
+        # self.model_.load_state_dict(
```
It is a hack for quickly test different llama size, but not for landing (because the actual checkpoint is missing).

lucylq commented:
@lucylq has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D52137894).

lucylq committed:
Update on "[flatbuffer] Use 7b params in Llama example"


Differential Revision: [D52137894](https://our.internmc.facebook.com/intern/diff/D52137894)

[ghstack-poisoned]
